src/Illuminate/Translation/FileLoader.php

```diff
@@ -114,6 +118,22 @@ protected function loadPath($path, $locale, $group)
     }
 
     /**
+     * Load a locale from a given JSON file.
+     *
+     * @param  string  $path
+     * @param  string  $locale
+     * @return array
+     */
+    protected function loadJson($path, $locale)
+    {
+        if ($this->files->exists($full = "{$path}/{$locale}.json")) {
+            return (array) json_decode($this->files->get($full));
```

Comment:
Can we use `json_decode($this->files->get($full), true)` instead? It'll give us an associative array [[1](http://php.net/json_decode)] :)